{
  "cve": {
    "data_type": "CVE",
    "data_format": "MITRE",
    "data_version": "4.0",
    "CVE_data_meta": {
      "ID": "CVE-2016-10700",
      "ASSIGNER": "cve@mitre.org"
    },
    "problemtype": {
      "problemtype_data": [
        {
          "description": [
            {
              "lang": "en",
              "value": "CWE-264"
            }
          ]
        }
      ]
    },
    "references": {
      "reference_data": [
        {
          "url": "https://web.archive.org/web/20160817090458/http://bugs.cacti.net/view.php?id=2697",
          "name": "https://web.archive.org/web/20160817090458/http://bugs.cacti.net/view.php?id=2697",
          "refsource": "CONFIRM",
          "tags": [
            "Issue Tracking",
            "Third Party Advisory"
          ]
        },
        {
          "url": "https://github.com/Cacti/cacti/commit/69983495cd41bf0903fe02baeef84b1fa85f2846",
          "name": "https://github.com/Cacti/cacti/commit/69983495cd41bf0903fe02baeef84b1fa85f2846",
          "refsource": "CONFIRM",
          "tags": [
            "Issue Tracking",
            "Patch",
            "Third Party Advisory"
          ]
        },
        {
          "url": "http://www.cacti.net/release_notes_1_0_0.php",
          "name": "http://www.cacti.net/release_notes_1_0_0.php",
          "refsource": "CONFIRM",
          "tags": [
            "Issue Tracking",
            "Release Notes",
            "Vendor Advisory"
          ]
        },
        {
          "url": "http://bugs.cacti.net/view.php?id=2697",
          "name": "http://bugs.cacti.net/view.php?id=2697",
          "refsource": "CONFIRM",
          "tags": [
            "Broken Link"
          ]
        }
      ]
    },
    "description": {
      "description_data": [
        {
          "lang": "en",
          "value": "auth_login.php in Cacti before 1.0.0 allows remote authenticated users who use web authentication to bypass intended access restrictions by logging in as a user not in the cacti database, because the guest user is not considered. NOTE: this vulnerability exists because of an incomplete fix for CVE-2016-2313."
        }
      ]
    }
  },
  "configurations": {
    "nodes": [
      {
        "operator": "OR",
        "cpe_match": [
          {
            "vulnerable": true,
            "cpe23Uri": "cpe:2.3:a:cacti:cacti:*:*:*:*:*:*:*:*"
          }
        ]
      }
    ]
  },
  "impact": {
    "baseMetricV2": {
      "cvssV2": {
        "vectorString": "AV:N/AC:L/Au:S/C:P/I:P/A:P",
        "accessVector": "NETWORK",
        "accessComplexity": "LOW",
        "authentication": "SINGLE",
        "confidentialityImpact": "PARTIAL",
        "integrityImpact": "PARTIAL",
        "availabilityImpact": "PARTIAL",
        "baseScore": 6.5
      },
      "severity": "MEDIUM",
      "exploitabilityScore": 8,
      "impactScore": 6.4,
      "obtainAllPrivilege": false,
      "obtainUserPrivilege": false,
      "obtainOtherPrivilege": false,
      "userInteractionRequired": false
    }
  },
  "publishedDate": "2017-11-24T05:29Z",
  "lastModifiedDate": "2017-12-11T19:35Z"
}